engine-tests/src/main/java/org/terasology/engine/testUtil/WithUnittestModule.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.testUtil;
+
+import org.terasology.engine.context.Context;
+import org.terasology.engine.core.GameEngine;
+import org.terasology.engine.core.module.ModuleManager;
+import org.terasology.engine.core.subsystem.EngineSubsystem;
+import org.terasology.gestalt.module.Module;
+
+public class WithUnittestModule implements EngineSubsystem {
+    @Override
+    public String getName() {
+        return "Unittest";
+    }
+
+    @Override
+    public void initialise(GameEngine engine, Context rootContext) {
+        EngineSubsystem.super.initialise(engine, rootContext);
```

Comment:
Calling `super` out of habit here, but the default implementation is a no-op. Is that a good practice or should we remove this `super`?

Reply:
I think I would not make a call to `super` when overriding _default_ methods of an interface usually :thinking:
I understand the `EngineSubsystem` interface as something where I get a bunch of hook methods I can choose to implement, but I don't have to. 

If it suddenly becomes necessary to always perform some kind of code here you'd probably change the underlying implementation such that contract stays the same for child classes (e.g., introduce a new "internal init method" that in turn calls the hook method...

tl;dr: I don't think it is necessary, and I would omit it.